config/runtime/server.go

```diff
@@ -129,15 +163,15 @@ func NewServerConfiguration(
 				config.NewAccessControl(srvConf.Spa.AccessControl, srvConf.Spa.DisableAccessControl), h)
 
 			for _, spaPath := range srvConf.Spa.Paths {
-				err = setRoutesFromHosts(serverConfiguration, serverOptions.ServerErrTpl, defaultPort, srvConf.Hosts, path.Join(serverOptions.SPABasePath, spaPath), spaHandler, spa)
+				err = setRoutesFromHosts(serverConfiguration, portsHosts, path.Join(serverOptions.SPABasePath, spaPath), spaHandler, spa)
 				if err != nil {
 					return nil, err
 				}
 			}
 		}
 
 		if srvConf.Files != nil {
-			fileHandler, err := handler.NewFile(serverOptions.FileBasePath, srvConf.Files.DocumentRoot, serverOptions)
+			fileHandler, err := handler.NewFile(serverOptions.FilesBasePath, srvConf.Files.DocumentRoot, serverOptions)
```

Comment:
This could be simplified.

```suggestion
			fileHandler, err := handler.NewFile(srvConf.Files.DocumentRoot, serverOptions)
```

Reply:
Done